debezium/ksql_queries.ksql

```diff
@@ -0,0 +1,84 @@
+{"ksql":
+
+"
+SET 'auto.offset.reset' = 'earliest';
+
+CREATE TABLE reference (
+    reference_id string PRIMARY KEY,
+    curie string,
+    abstract string,
+    category string,
+    date_arrived_in_pubmed string,
+    date_created bigint,
+    date_last_modified_in_pubmed string,
+    date_published string,
+    date_updated bigint,
+    issue_name string,
+    keywords array<string>,
+    language string,
+    open_access boolean,
+    page_range string,
+    plain_language_abstract string,
+    publisher string,
+    pubmed_abstract_languages string,
```

Comment:
This should probably be array<string>  
in sql     pubmed_abstract_languages character varying[] COLLATE pg_catalog."default",